tests/foreman/ui/test_ansible.py

```diff
@@ -38,6 +38,10 @@ def test_positive_create_delete_variable_with_overrides(
 
         :id: 90acea37-4c2f-42e5-92a6-0c88148f4fb6
 
+        :customerscenario: true
+
+        :BZ: 2181595
```

Comment:
We can also add jira
```suggestion
        :Verifies: SAT-19502
```